src/apply-names.h

```diff
@@ -36,8 +38,8 @@ struct WabtModule;
  *    ...
  *    (call $foo ...)
  */
-WABT_EXTERN_C_BEGIN
-WabtResult wabt_apply_names(struct WabtModule*);
-WABT_EXTERN_C_END
+Result apply_names(struct Module*);
```

Comment:
I was thinking since we're using C++ features in the headers now, I'd drop all of the extern "C" stuff. When we get back to place where we're happy with the C++ API, we can add back in a nice C wrapper API